doc/docs/master/how-tos/advanced-data-operations/use-transactions-to-run-multiple-commands.md

```diff
@@ -4,19 +4,164 @@
+          You must finish your transaction before putting files in the corresponding repo for the data to be 
+          part of the same batch. Running a 'put file' before closing the transaction would result in a commit being created 
+          independently from the transaction itself and a job to run on that commit.
+
+## Supported Operations
+
+While there is a transaction object in the Pachyderm configuration
+file, all supported API requests append the request to the
+transaction instead of running directly. These supported commands include:
+
+```shell
+create repo
+delete repo
+update repo
+start commit
+finish commit
+squash commitset
+create branch
+delete branch
+create pipeline
+update pipeline
+edit pipeline
+```
+
+Each time you add a command to a transaction, Pachyderm validates the
+transaction against the current state of the cluster metadata and obtains
+any return values, which is important for such commands as
+`start commit`. If validation fails for any reason, Pachyderm does
+not add the operation to the transaction. If the transaction has been
+invalidated by changing the cluster state, you must delete the transaction
+and start over, taking into account the new state of the cluster.
+From a command-line perspective, these commands work identically within
+a transaction as without. The only difference is that you do not apply
+your changes until you run `finish transaction`, and a message that
+Pachyderm logs to `stderr` to indicate that the command was placed
+in a transaction rather than run directly.
+
+## Other Transaction Commands
+Other supporting commands for transactions include the following commands:
+
+| Command      | Description |
+| ------------ | ----------- |
+| pachctl list transaction| List all unfinished transactions available in the Pachyderm cluster. |
+| pachctl stop transaction | Remove the currently active transaction from the local Pachyderm config file. The transaction remains in the Pachyderm cluster and can be resumed later. |
+| pachctl resume transaction | Set an already-existing transaction as the active transaction in the local Pachyderm config file. |
+| pachctl delete transaction | Deletes a transaction from the Pachyderm cluster. |
+| pachctl inspect transaction | Provides detailed information about an existing transaction, including which operations it will perform. By default, displays information about the current transaction. If you specify a transaction ID, displays information about the corresponding transaction. |
+
+## Multiple Opened Transactions
+
+Some systems have a notion of *nested* transactions. That is when you
+open transactions within an already opened transaction. In such systems, the
+operations added to the subsequent transactions are not executed
+until all the nested transactions and the main transaction are closed.
```

Comment:
I wonder if it makes sense to stick with "finished" here as above to avoid confusion with closing the commits